### ISSUE TITLE
nb-card-header doesn't render in initial load

### ISSUE THREAD

**Issue by dogukanevcil** [akveo/nebular#2755]
**I'm submitting a bug report** 

### Issue description

**Current behavior:**
When I login and redirect to the authenticated part of my application. I can't see `nb-card-header` until I refresh the page. The `nb-card-header` component can be seen in inspect even if it is not rendered. 

**Before refresh (initial load)**
![GP3tP](https://user-images.githubusercontent.com/36449961/120311709-7b2de400-c2e0-11eb-83fc-5ce5273e002d.png)

**After refresh**
![7pfoV](https://user-images.githubusercontent.com/36449961/120311660-6c473180-c2e0-11eb-9390-cb4118e2702f.png)

This is happening when a user signs in and redirected to the dashboard. In dashboard I have multiple pages like this and none of them render `nb-card-header`. When I navigate site through dashboard with `router`, it doesn't render aswell but if I reload one of the pages or navigate by changing url it starts to render. 

I can see `nb-card-header` component in the inspect even if it is not rendered. 

![EMYhd](https://user-images.githubusercontent.com/36449961/120312458-67cf4880-c2e1-11eb-9db9-30294f40dff2.png)

**Expected behavior:**
It should render initially like other components. 

**Related code:**

```
    <nb-card [nbSpinner]="loading">
      <nb-card-header>
        Users
        <button nbButton
                [routerLink]="['/users/new']"
                class="table-button"
                status="primary"
                size="small">
          NEW
          <nb-icon icon="plus-outline"></nb-icon>
        </button>

      </nb-card-header>
      <nb-card-body>
        <table datatable
               [dtOptions]="dtOptions"
               [dtTrigger]="dtTrigger"
               width="100%"
               cellspacing="0"
               class="row-border hover">
          <thead>
          <tr>
            <th>ID</th>
            <th>Name</th>
            <th>E-mail</th>
            <th>Organisation</th>
            <th>Actions</th>
          </tr>

          ....

**Comment by Jannexy** [comment]
Did you resolve this issue?
I have the same problem and I don't found the solution

**Comment by dogukanevcil** [comment]
Yes, I did a workaround. I had an login component where I redirect the user to the dashboard after login.  As the headers are visible after refresh, I used `window.location.href` in the login component for the redirect.
